docs/examples/hosting/docker/basic/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.7"
+services:
+    server:
+        image: padloc/server
+        build:
+            context: github.com/padloc/padloc.git#docs/docker-examples
```

Comment:
We'll have to remember to changes the ref to `v4` before merging, and to remove it before merging into master.